ocaml/xapi/xapi_pool.ml

```diff
@@ -3345,10 +3346,28 @@ let enable_tls_verification ~__context =
   | Some self ->
       Xapi_cluster_host.set_tls_config ~__context ~self ~verify:true
 
+let assert_enable_single_bundle_repo ~__context ~repos =
+  if
+    List.length repos > 1
+    && List.exists
+         (fun repo -> Db.Repository.get_origin ~__context ~self:repo = `bundle)
+         repos
+  then
+    raise Api_errors.(Server_error (bundle_repo_should_be_single_enabled, []))
```

Comment:
When this happened, it would be an internal bug.

Reply:
It's not an internal bug. This exception is raised by function `assert_enable_single_bundle_repo`. It checks if the bundle repo is the only one enabled repo when executing `set_repositories` or `add_repositories`.